src/firmware/app/control/wheel_controller.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include "firmware/shared/circular_buffer.h"
+
+// wheel_controller_difference_equation.h
+typedef struct WheelController
```

Comment:
Shouldn't this struct be defined in the `.c` file? (to hide the internals like we do for pseudoclasses)

Reply:
Moved to the .c 👍